source/crud/tests/unified/aggregate-write-readPreference.yml

```diff
@@ -87,7 +87,8 @@ tests:
 
   - description: "Aggregate with $out omits read preference for pre-5.0 server"
     runOnRequirements:
-      - maxServerVersion: "4.4.99"
+      - minServerVersion: "4.2"
```

Comment:
[opt] You could leave a small comment here and on the db test yml explaining why we're adding the `minServerVersion`.